src/popover/popover.ts

```diff
@@ -156,6 +156,14 @@ export class NgbPopover implements OnInit, OnDestroy, OnChanges {
    * @since 2.2.0
    */
   @Input() popoverClass: string;
+  /**
+   * Delay (in ms) before opening the popover after a non-manual opening trigger.
+   */
+  @Input() triggerOpenDelay = 0;
```

Comment:
- these inputs should come from `NgbPopoverConfig`
- related tests are in `ngb-popover-config.ts`